asserts/fetcher.go

```diff
@@ -50,6 +51,28 @@ func NewFetcher(trustedDB RODatabase, retrieve func(*Ref) (Assertion, error), sa
 	}
 }
 
+// FIXME: move to asserts.go and make public?
+func newRef(uniq string) *Ref {
+	l := strings.Split(uniq, "/")
+	if len(l) < 2 {
+		return nil
+	}
+	return &Ref{
+		Type:       Type(l[0]),
+		PrimaryKey: l[1:],
+	}
+}
+
+func (f *Fetcher) SavedRefs() []*Ref {
```

Comment:
this is a strange API for Fetcher given that is all about doing things in the right order, and this returns things out of order